src/plugins/search/plugin.search.js

```diff
@@ -45,6 +45,13 @@ jQuery.extend(BookReader.defaultOptions, {
 /** @override */
 BookReader.prototype.setup = (function (super_) {
   return function (options) {
+    // override with config
+    if (options.enableSearch === false) {
+      jQuery.extend(BookReader.defaultOptions, {
+        enableSearch: false
+      });
+    }
```

Comment:
This is changing the default bookreader options if the user option is set to false; are we sure this is needed?

Reply:
hi drini, thanks for asking.. the problem i am trying to solve is - that `enableSearch` flag that gets set [above on line 40] (https://github.com/internetarchive/bookreader/blob/master/src/plugins/search/plugin.search.js#L35-L43) - when the search plugin script is loaded.  This overrides any config sets when BR gets loaded.

the check in the constructor uses the received value to see if it needs to be turned off.

the only other option that i cannot avail is to _not_ load the plugin script.

Reply:
The plugins are loaded and modify the default options, `BookReader.defaultOptions`. These are then overriden inside the bookreader contructor:

https://github.com/internetarchive/bookreader/blob/3755718097e0023609c0615553099f8572c4c568/src/BookReader.js#L57

So they _should_ always be happening in the right order?